playground/mocks/data/idp/idx/error-authenticator-enroll-passcode-change-on-prem.json

```diff
@@ -9,7 +9,7 @@
       {
         "message": "Wait for token to change, then enter the new tokencode.",
         "i18n": {
-          "key": "oie.on_prem.error.token.change",
+          "key": "errors.E0000106",
```

Comment:
I'd suggest we use an `oie.*` value here (or transform it), as we should try to avoid grouping certain authenticator error responses to our global error.

From a customization standpoint, this helps us be as clear as possible on keys that are mapped directly to API responses.

Reply:
I think the goal is to re-use an existing V1 error key ?
```
errors.E0000106 = Wait for token to change, then enter the new tokencode.
```

Reply:
Yes, @nikhilvenkatraman-okta is right. I was trying to use our existing error override string